libcloud/compute/drivers/vcloud.py

```diff
@@ -1044,8 +1063,6 @@ def ex_deploy_node(self, node):
                                       method='POST',
                                       headers=headers)
         self._wait_for_task_completion(res.object.get('href'))
-        res = self.connection.request(get_url_path(node.id))
```

Comment:
It's intentional. That's an internal method that handles both vapp's and vm's. It's meant to be called only from ex_deploy_node (where the node is indeed returned).